package.json

```diff
@@ -38,12 +41,11 @@
     "prebuild:color-tokens": "rm -rf tokens-v2-private/**/**/color",
     "prepack": "yarn build && yarn build:tokens",
     "prepare": "husky install",
+    "postinstall": "npm run install:storybook",
```

Comment:
eek.. this will also install for users of the npm package. 10/10 don't recommend. 

can you put this into the prepare command instead?

Reply:
Ahhh, okay, I didn't know `prepare` runs on local npm install as well. 👍  will change it.